waspc/src/Wasp/Generator/Common.hs

```diff
@@ -34,3 +39,17 @@ npmVersionRange = SV.Range [SV.backwardsCompatibleWith latestLTSVersion]
 
 prismaVersion :: SV.Version
 prismaVersion = SV.Version 4 5 0
+
+makeJsonWithEntityNameAndPrismaIdentifier :: String -> Aeson.Value
+makeJsonWithEntityNameAndPrismaIdentifier name =
+  object
+    [ "name" .= name,
+      "prismaIdentifier" .= entityNameToPrismaIdentifier name
+    ]
```

Comment:
Renamed, as requested in https://github.com/wasp-lang/wasp/pull/945#discussion_r1082488852.